docs/onboarding/Configuration-Properties.md

```diff
@@ -142,6 +142,7 @@ The Onboarding Server uses the following public configuration properties:
 | `enrollment-server-onboarding.provider.innovatrics.serviceBaseUrl`                               |                           | Base REST service URL for Innovatrics.                                         |
 | `enrollment-server-onboarding.provider.innovatrics.serviceToken`                                 |                           | Authentication token for Innovatrics.                                          |
 | `enrollment-server-onboarding.provider.innovatrics.serviceUserAgent`                             | `Wultra/OnboardingServer` | User agent to use when making HTTP calls to Innovatrics REST service.          |
+| `enrollment-server-onboarding.provider.innovatrics.presenceCheck.score`                          | 0.875                     | Presence check minimal score threshold.                                        |
```

Comment:
Is `0.875` a reasonable default?

Reply:
See the link https://developers.innovatrics.com/digital-onboarding/docs/functionalities/face/active-liveness-check/#magnifeye-liveness It is included in the documentation below.